package.json

```diff
@@ -179,7 +179,7 @@
     },
     "scripts": {
         "vscode:prepublish": "del-cli dist/template && webpack --mode production && cd ./src/template && node ../../node_modules/copy/copy.js ./**/* ../../dist/template",
-        "vscode:package": "node ./node_modules/vsce/vsce package",
+        "vscode:package": "node ./node_modules/@vscode/vsce/vsce package",
```

Comment:
💡Just idea: unless we need this we can remove this line throughout. Let’s merge this for now.

Looks good to me.